### ISSUE TITLE
[Feature] Not-persistent context on Android device

### ISSUE THREAD

**Issue by gskierk** [microsoft/playwright#24067]
Let us know what functionality you'd like to see in Playwright and what your use case is.

I'd like my tests on an Android device to be fully isolated the same way they are when running on a desktop browser. In this particular case, the context is persistent, and clearing the cookies and closing all pages programmatically may not be enough (notice that there are also local, session, and other storages...). I'd be happy if I could launch the browser in isolation, e.g., in incognito mode.

TLDR: I need test isolation on Android device.

Do you think others might benefit from this as well?

Yes.

**Comment by laelhalawani** [comment]
We could really use this functionality too.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
